src/telliot_feeds/reporters/rewards/time_based_rewards.py

```diff
@@ -8,19 +9,28 @@
 
 async def get_time_based_rewards(oracle: Tellor360OracleContract) -> int:
     """
-    Time based rewards, as of Tellor360,
-    are the TRB balance in the Oracle contract
-    minus the TRB staking rewards and
-    the totalStakeAmount
+    Reward that will be given if a reporter submits now:
+    TBR is calculated by factoring in the time of the last value to determine when it
+    started accruing and the total rewards balance available in the contract since thats
+    what will be dispersed if the reward is < than whats available in the contract
     """
-
     tbr: int
+    reward: int
+
+    time_of_last_new_value, last_val_status = await oracle.read("timeOfLastNewValue")
+    tbr, tbr_status = await oracle.read("getTotalTimeBasedRewardsBalance")
 
-    tbr, status = await oracle.read("getTotalTimeBasedRewardsBalance")
+    # if any call fails return 0 and a msg that tbr can't calc'd
+    if not tbr_status.ok or not last_val_status.ok:
+        error = tbr_status.error if not tbr_status.ok else last_val_status.error
+        logger.warning(f"Unable to calculate time-based rewards for reporter: {error}")
+        return 0
 
-    if status.ok:
-        return tbr
+    reward = (
+        (TimeStamp.now().ts - time_of_last_new_value) * 5e17
+    ) / 300  # amount of TRB rewards released per 5 minutes
 
-    else:
-        logger.warning("Unable to calculate time-based rewards for reporter: " + status.error)
+    if tbr <= 0 and reward <= 0:
```

Comment:
When would either be less than zero?

Reply:
at the beginning of the contract is one instance where they could both be zero, or when someone depletes the contract and no one mints to oracle.

Reply:
But less than zero?

If there isn't a case where either are less than zero, you could just get rid of this line. And line 36 would be enough bc it would return zero, no?

Reply:
I'm saying idk if there isn't a case.  but the contract probably handles underflow so will remove.